src/common/util/firebase.ts

```diff
@@ -219,3 +225,33 @@ export function indexParsedSentence(sentence: string, tokens: string[], js: stri
     })
     .catch(noop);
 }
+
+export async function getRecentGamesByUserId(userId: string): Promise<Array<object>> {
+  const blueGames = await getRecentGamesForColorByUserId(userId, 'blue');
+  const orangeGames = await getRecentGamesForColorByUserId(userId, 'orange');
+
+  return Object.values({...blueGames, ...orangeGames});
+}
+
+async function getRecentGamesForColorByUserId(userId: string, color: string): Promise<object> {
+  const snapshot = await fb.database()
+    .ref('games')
+    .orderByChild(`players/${color}`)
+    .equalTo(userId)
+    .once('value');
+  return snapshot.val();
+}
+
+export async function getCardsCreatedCountByUserId(userId: string): Promise<number> {
```

Comment:
nit: maybe this method and the one below could somehow be deduplicated? I guess Rule of Three says don't bother until we have another similar method?

Reply:
Hmm yeah I was thinking what a good name for the shared method would be